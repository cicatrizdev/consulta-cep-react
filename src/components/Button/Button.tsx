import React from 'react'
import styled from 'styled-components'

const Wrapper = styled.button`
  border-radius: 6px;
  background: #7e7aec;
  color: white;
  cursor: pointer;
  font-size: 1rem;
  padding: .35rem 1rem;
  
  &:hover {
    border: 1px solid #7e7aec;
  }
`

const Button = () =>
  <Wrapper>
    Buscar
  </Wrapper>

export default Button;