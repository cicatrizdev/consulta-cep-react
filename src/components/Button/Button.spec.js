import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';

import Button from './Button'

describe('App', () => {
  test('snapshot renders', () => {
    const component = renderer.create(<Button />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});