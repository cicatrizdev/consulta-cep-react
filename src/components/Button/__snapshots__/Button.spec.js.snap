// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot renders 1`] = `
<button
  className="sc-bdVaJa faugNV"
>
  Buscar
</button>
`;
